### PR TITLE
adding tests for wish ORC

### PR DESCRIPTION
**Pull request by rerpha**

### CODE REVIEW

**Review by DominicOram — CHANGES_REQUESTED** [review]
Sorry, I know we talked about this already but I actually think it would be good to have them both using a shared file as there is quite a bit of repetition. Also, can we have some tests for the logic of working out the time until the next maintenance roattion?

File: tests/oscillating_collimator_wish.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    Tests for the LET Oscillating collimator.
+
+    The CA.Client.Exceptions these tests generate are expected because of a workaround we had to make in the DB
+    file to prevent a hang in the case of using asynFloat64 for the SP types. Issue described in ticket #2736
+    """
+    def setUp(self):
+        self._ioc = IOCRegister.get_running("GALIL_01")
+        ca_mot = ChannelAccess()
+        ca_mot.assert_that_pv_exists("MOT:MTR0101", timeout=30)
+        self.ca = ChannelAccess(device_prefix=PREFIX)
```
[the remark this answers is not in the document]
Comment (rerpha) [reply tests/oscillating_collimator_wish.py]: They pass, it only speeds it up 10 times 😆

File: tests/oscillating_collimator_wish.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        self.ca.set_pv_value(FREQUENCY, -1.0)
+        self.ca.assert_that_pv_is_number(FREQUENCY, 0.0)
+
+    def test_WHEN_angle_set_greater_than_half_THEN_angle_is_half(self):
+        self.ca.set_pv_value(FREQUENCY, 1.0)
+        self.ca.assert_that_pv_is_number(FREQUENCY, 0.5)
+
+    def test_WHEN_frq_set_greater_than_two_THEN_angle_is_two(self):
+        self.ca.set_pv_value(ANGLE, 5.0)
+        self.ca.assert_that_pv_is_number(ANGLE, 2.0)
```
[the remark this answers is not in the document]
Comment (rerpha) [reply tests/oscillating_collimator_wish.py]: yes they are, im not sure they are actually worth keeping regardless as the discriminant calc stuff is all covered in the parametrised test anyway

**Review by DominicOram — APPROVED** [review]
Great, thank you

**Pull request merged**